scripts/python_installer

```diff
@@ -105,10 +107,15 @@ function install_python() {
     echo_success_message " - Python $PYTHON_VERSION is installed at $PYENV_BIN"
 
     if python_is_not_in_path; then
-        print_path_export_instructions
+        if ${SHOW_PATH_EXPORT_MESSAGE} ; then
```

Comment:
You can combine the two `if`s into a multi-operand conditional.

Reply:
BTW, the default behaviour of this file must be to show rather than suppress. The prerogative to show the Python-`PATH`-export message must be that of the `python_installer`. The only right that an external entity (such as the `bundled_installer`) may be afforded is that of "requesting" suppression.

Given that, we must rename this variable to `SUPPRESS_PATH_EXPORT_MESSAGE` and set its default value to `false along the lines of what we had discussed [here](https://github.com/aws/aws-elastic-beanstalk-cli-setup/pull/54#discussion_r318889122).

Reply:
I will change the variable name to be more descriptive. 

To your point about the multi-operand conditional, I think the way it is above is suitable for out use. If I combine the the two checks into one, then it would be checking for both python not being in path and if I want to suppress the export message. In the case of wanting to call the print_path_export_instructions, this is the correct usage. However, this misses the case where python is not in path but I do want to suppress this message. In that case I want the script to do nothing. The else statement below it only handles the situation where python is in path and we do want to show or hide the message. The solution here is to either add another case to this if or leave it as is. I think adding an elif would be a little more verbose.